web/src/main/java/config/YggdrasilWebConfig.java

```diff
@@ -27,6 +27,7 @@ public void addResourceHandlers(ResourceHandlerRegistry registry) {
     public InternalResourceViewResolver viewResolver() {
         InternalResourceViewResolver viewResolver = new InternalResourceViewResolver();
         viewResolver.setViewClass(JstlView.class);
+        viewResolver.setContentType("text/html; charset=utf-8");
```

Comment:
It doesn't resolve our issue with encoding, so maybe remove this until we need it?